share/doc/homebrew/Prose-Style-Guidelines.md

```diff
@@ -0,0 +1,77 @@
+
+The intent of this document is to help authors make decisions about clarity, style, and consistency. It is not to help settle arguments about who knows English better. Don't use this document to be a jerk.
+
+## Guidelines
+
+We prefer:
+
+### Style and usage
+
+* British/Commonwealth English over American English, in general
+* "e.g." and "i.e.": Go ahead and use "e.g." or "i.e." instead of spelling them out. Don't worry about putting a comma after them.
+* Offset nontrivial subordinate clauses with commas
+
+### Personal pronouns
+
+* We respect all people's choice of personal pronouns
+* Singular "they" when someone's gender is unknown
+* Avoid gender-specific language when not necessary
+
+### Structure and markup
+
+* Sentence case in section headings, not Title Case
+* Periods at the ends of list items where most items in that list are complete sentences
+* More generally, parallel list item structure
+* Capitalize all list items if you want, even if they're not complete sentences; just be consistent within each list, and preferably, throughout the whole page
+* Use a subordinate list item instead of dropping a multi-sentence paragraph-long item into a list of sentence fragments
+* GitHub flavored Markdown. GitHub's implementation is the standard, period.
+
+### Typographical conventions
+
+* Literal text in commands and code is styled in `fixed width font`
+* Placeholders inside code snippets are marked up with `<...>` brackets
+ * e.g. `git remote add <my-user-name> https://github.com/<my-user-name>/homebrew-core.git`
+* No "$" with environment variables mentioned outside code snippets
+ * e.g. "Set `BLAH` to 5", not "Set `$BLAH` to 5"
+* One space after periods, not two
+* Capitalized proper nouns
+* We do not defer to extensive nonstandard capitalization, typesetting, or other styling of brand names, aside from the normal capitalization of proper nouns and simple internal capitalization
+* No "TM", &trade;, <sup>SM</sup>, &copy;, &reg;, or other explicit indicators of rights ownership or trademarks; we take these as understood when the brand name is mentioned
+
+### Terminology, words, and word styling
+
+* "pull request", not "Pull Request"
+* "check out" is the verb; "checkout" is the noun
+* Spell out certain technical words
+ * "repository", not "repo"
+ * When abbreviating, introduce the abbreviation with the first usage in any document
+* "OS X" or "Mac OS X", not "OSX" or "MacOS"
+* "RuboCop", not "Rubocop"
+* A pull request is made "on" a repository; that repository is "at" a URL
+
+## How to use these guidelines
+
+Refer to these guidelines to make decisions about style and usage in your own writing for Homebrew documents and communications.
```

Comment:
I think `communication` may be a little easier here; the plural isn't quite the same thing in all contexts. You could use `communique` I guess but that's probably hostile to the purpose of the document.